deerlab/fitsignal.py

```diff
@@ -15,7 +14,8 @@
 from deerlab.utils import isempty, goodness_of_fit, Jacobian
 
 def fitsignal(Vexp, t, r, dd_model='P', bg_model=bg_hom3d, ex_model=ex_4pdeer,
-              par0=[None,None,None], lb=[None,None,None], ub=[None,None,None], verbose= False,
+              par0_dd=None, par0_bg=None, par0_ex=None, verbose= False,
+              lb_dd=None, lb_bg=None, lb_ex=None, ub_dd=None, ub_bg=None, ub_ex=None,

```

Comment:
Wouldn't it be better to call these arguments `dd_par0` instead of `par0_dd`, etc? There is `dd_model`, so `dd_par0`, `dd_lb`, and `dd_ub` would be more consistent.

Reply:
Agreed